lib/divo/compose.ex

```diff
@@ -61,8 +61,10 @@ defmodule Divo.Compose do
 
   defp log_compose({message, 0}), do: Logger.info(message)
 
-  defp log_compose({message, code}),
-    do: Logger.error("Docker Compose exited with code: #{code}. #{message}")
+  defp log_compose({message, code}) do
+    Logger.error("Docker Compose exited with code: #{code}. #{message}")
+    raise "Docker Compose exited with code: #{code}. #{message}"
```

Comment:
I could see the argument for running a specific ErrorType, but if the default is RunTimeError or similar I like it.

Reply:
Talked in person, sticking with the generic runtime error